### PR TITLE
NV UP Migration Receiver Translation and settings

### PR DESCRIPTION
**Pull request by oslynn**
This PR adds Nevada UP receiver transform and settings.

Test Steps:
1. *Include steps to test these changes*

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

### Testing
- [ ] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

**Review by GilmoreA6 — CHANGES_REQUESTED** [review]
Can you add integration tests for the transform?

File: prime-router/settings/prod/0163-nv-prod-up-onboarding.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  receivers:    
+  - name: "full-elr"
+    organizationName: "nv-phd"
+    topic: "full-elr"
+    customerStatus: "inactive"
+    translation: !<HL7>
+      schemaName: "metadata/hl7_mapping/receivers/STLTs/NV/NV-receiver-transform"
+      useTestProcessingMode: false
+      useBatchHeaders: true
+      receivingApplicationName: "NVDOH"
```
Comment (adegolier) [inline prime-router/settings/prod/0163-nv-prod-up-onboarding.yml:19]: Doesn't this go into the transform?

File: prime-router/settings/prod/0163-nv-prod-up-onboarding.yml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      receivingApplicationOID: null
+      receivingFacilityName: "NVODH"
```
Comment (adegolier) [inline prime-router/settings/prod/0163-nv-prod-up-onboarding.yml:21]: Doesn't this also go into the transform? And is it supposed to be ODH?

File: prime-router/settings/prod/0163-nv-prod-up-onboarding.yml (hunk 1 above)
Comment (oslynn) [reply prime-router/settings/prod/0163-nv-prod-up-onboarding.yml:19]: @adegolier, thank you and yes, I have it in the transform as well.  I am updating as following:
image

File: prime-router/settings/prod/0163-nv-prod-up-onboarding.yml (hunk 2 above)
Comment (oslynn) [reply prime-router/settings/prod/0163-nv-prod-up-onboarding.yml:21]: Yes, you are correct as above.

File: prime-router/settings/prod/0163-nv-prod-up-onboarding.yml [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      reportingFacilityName: null
+      reportingFacilityId: null
+      reportingFacilityIdType: null
+      suppressQstForAoe: false
+      suppressHl7Fields: null
+      suppressAoe: true
+      defaultAoeToUnknown: false
+      replaceUnicodeWithAscii: false
+      useBlankInsteadOfUnknown: null
+      truncateHDNamespaceIds: true
```
Comment (adegolier) [inline prime-router/settings/prod/0163-nv-prod-up-onboarding.yml:39]: Ahh I missed these other couple when I was looking before... I think this also needs to go in the transform?

File: prime-router/settings/prod/0163-nv-prod-up-onboarding.yml [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        MSH-4-1:
+        - '*': "CDC PRIME"
+        MSH-4-2:
+        - '*': "11D2030855"
+      reportingFacilityName: null
+      reportingFacilityId: null
+      reportingFacilityIdType: null
+      suppressQstForAoe: false
+      suppressHl7Fields: null
+      suppressAoe: true
```
Comment (adegolier) [inline prime-router/settings/prod/0163-nv-prod-up-onboarding.yml:35]: Also goes in the transform?

File: prime-router/settings/prod/0163-nv-prod-up-onboarding.yml (hunk 3 above)
Comment (victor-chaparro) [reply prime-router/settings/prod/0163-nv-prod-up-onboarding.yml:39]: We do support this setting in the UP, but it does need to be tested.

**Pull request closed without merging**